godtools/ViewControllers/Tract/TractViewController.swift

```diff
@@ -8,13 +8,45 @@
 
 import Foundation
 import UIKit
+import SWXMLHash
 
 class TractViewController: BaseViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
         displayTitle()
+        initializeView()
+        setupSwipeGestures()
+    }
+    
+    func setupSwipeGestures() {
+        let swipeLeft = UISwipeGestureRecognizer(target: self, action: #selector(handleGesture))
+        swipeLeft.direction = .left
+        self.view.addGestureRecognizer(swipeLeft)
+        
+        let swipeRight = UISwipeGestureRecognizer(target: self, action: #selector(handleGesture))
+        swipeRight.direction = .right
+        self.view.addGestureRecognizer(swipeRight)
+    }
+    
+    @objc fileprivate func handleGesture(gesture: UISwipeGestureRecognizer) {
+        if gesture.direction == .right {
+            moveToPreviousPage()
+        } else if gesture.direction == .left {
+            moveToNextPage()
+        }
+    }
+    
+    func moveToNextPage() {
```

Comment:
So I was thinking there would be one view controller that would hold a collection of views. Swiping to the left or right would replace the current view with the next or previous. It would be similar, I think to how you build then onboarding flow.